### ISSUE TITLE
Work-around to "RuntimeError... 'float' object cannot be interpreted as an integer during execution" for kcc-c2e.py powershell/CLI

### ISSUE THREAD

**Issue by aaronshiu** [ciromattia/kcc#406]
Hope this is helpful, but I tried implementing the program running kcc-c2e.py for a directory/cbz archive file with powershell on windows. During execution, I kept coming across: 

Processing images...
Traceback (most recent call last):
  File "<Directory>\kcc\kcc-c2e.py", line 32, in <module>
    startC2E()
  File "<Directory>\kcc\kindlecomicconverter\startup.py", line 49, in startC2E
    sys.exit(main(sys.argv[1:]))
  File "<Directory>\kcc\kindlecomicconverter\comic2ebook.py", line 74, in main
    makeBook(source)
  File "<Directory>\kcc\kindlecomicconverter\comic2ebook.py", line 1089, in makeBook
    imgDirectoryProcessing(os.path.join(path, "OEBPS", "Images"))
  File "<Directory>\kcc\kindlecomicconverter\comic2ebook.py", line 526, in imgDirectoryProcessing
    raise RuntimeError("One of workers crashed. Cause: " + workerOutput[0][0], workerOutput[0][1])
RuntimeError: ("One of workers crashed. Cause: 'float' object cannot be interpreted as an integer", '  File "<Directory>\\kcc\\kindlecomicconverter\\comic2ebook.py", line 563, in imgFileProcessing\n    img.autocontrastImage()\n  File "<Directory>\\kcc\\kindlecomicconverter\\image.py", line 258, in autocontrastImage\n    self.image = ImageOps.autocontrast(Image.eval(self.image, lambda a: 255 * (a / 255.) ** gamma))\n  File "C:\\Program Files\\Python310\\lib\\site-packages\\PIL\\Image.py", line 3107, in eval\n    return image.point(args[0])\n  File "C:\\Program Files\\Python310\\lib\\site-packages\\PIL\\Image.py", line 1680, in point\n    return self._new(self.im.point(lut, mode))\n')

For kcc\kindlecomicconverter\image.py, on line 258, 
this was changed from: 255 * (a / 255.) ** gamma
to: int(255 * (a / 255.) ** gamma)

So the line became: self.image = ImageOps.autocontrast(Image.eval(self.image, lambda a: int(255 * (a / 255.) ** gamma)))

This resolved the issue and allowed the program to run. :)

**Comment by ScrambleTheRamble** [comment]
Just tried this fix but all my pages are black? Does this occur for you too?

**Comment by aaronshiu** [comment]
> Just tried this fix but all my pages are black? Does this occur for you too?

Hmm.. It seems alright for me when I open up converted files with Calibre e-book reader. It might have to do with how non-kindle programs display the black-white pages? Maybe you could ask the repo owner. :)

**Comment by Carleslc** [comment]
I am having the same issue with Python 3.10. The last working version seems to be 3.9.9.

Maybe a pull request with the solution provided here can fix this issue? I don't know if there are better solutions though.

_line 258 kcc/kindlecomicconverter/image.py_
```python
self.image = ImageOps.autocontrast(Image.eval(self.image, lambda a: int(255 * (a / 255.) ** gamma)))
```

A new minor release would be great too so it can be used via pip install on other projects. Currently a downgrade may be acceptable, but in the future may be a blocker to use some Python language features in version 3.10 and beyond.

**Comment by koroki** [comment]
> I am having the same issue with Python 3.10. The last working version seems to be 3.9.9.
> 
> Maybe a pull request with the solution provided here can fix this issue? I don't know if there are better solutions though.
> 
> _line 258 kcc/kindlecomicconverter/image.py_
> 
> ```python
> self.image = ImageOps.autocontrast(Image.eval(self.image, lambda a: int(255 * (a / 255.) ** gamma)))
> ```
> 
> A new minor release would be great too so it can be used via pip install on other projects. Currently a downgrade may be acceptable, but in the future may be a blocker to use some Python language features in version 3.10 and beyond.

Thanks for it! At least the "path" is working for me.

**Comment by ScrambleTheRamble** [comment]
@aaronshiu The fix works with pillow 8.4.0, not with 8.3.2 (which is installed as an rpm on Fedora 35).

**Comment by darodi** [comment]
- fixed in #457
